model/src/cfd.rs

```diff
@@ -296,6 +298,35 @@ impl Order {
             self.opening_fee,
         )
     }
+
+    const OUTDATED_AFTER_MINS: i64 = 10;
+
+    /// Defines when we consider the order to be sane within the system boundaries
+    ///
+    /// This is a safety net for the taker to prevent the taker from taking an outdated order.
```

Comment:
🔧 A bit simpler?

```suggestion
    /// Defines when we consider the order to be outdated.
    ///
    /// This is used as a safety net to prevent the taker from taking an outdated order.
```